kfs-core/src/main/java/edu/arizona/kfs/fp/batch/service/impl/ProcurementCardCreateDocumentServiceImpl.java

```diff
@@ -118,13 +145,13 @@ public boolean rerouteProcurementCardDocuments() {
               //step two, reroute documents enroute to the reconciler group(s) that had group membership problems
                 else if (node.equals("ProcurementCardReconciler") && groupRerouting && 
                         parameterService.getParameter(ProcurementCardRerouteDocumentsStep.class,
-                                RECONCILER_GROUPS_TO_REROUTE_PARM_NM).getValue() != null
+                                KFSParameterKeyConstants.ProcurementCardParameterConstants.RECONCILER_GROUPS_TO_REROUTE_PARM_NM).getValue() != null
```

Comment:
See note above, the two conditions in the if can be refactored to:
.. ** StringUtils.isNotEmpty(reconcilerGroups) && reconcilerGroups.contains(....) etc.